src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -227,15 +237,67 @@ class WelcomePage extends Disposable {
 			name: localize('welcome.title', "Welcome"),
 			resource,
 			telemetryFrom,
-			onReady: (container: HTMLElement) => this.onReady(container, recentlyOpened, installedExtensions, fileService)
+			onReady: (container: HTMLElement) => this.onReady(container, recentlyOpened, installedExtensions, fileService, layoutService)
 		});
 	}
 	public openEditor() {
 		return this.editorService.openEditor(this.editorInput, { pinned: false });
 	}
-	private onReady(container: HTMLElement, recentlyOpened: Promise<IRecentlyOpened>, installedExtensions: Promise<IExtensionStatus[]>, fileService: IFileService): void {
+	private onReady(container: HTMLElement, recentlyOpened: Promise<IRecentlyOpened>, installedExtensions: Promise<IExtensionStatus[]>, fileService: IFileService, layoutService: ILayoutService, ): void {
 		const enabled = isWelcomePageEnabled(this.configurationService, this.contextService);
 		const showOnStartup = <HTMLInputElement>container.querySelector('#showOnStartup');
+		const guidedTourEnabled = isGuidedTourEnabled(this.configurationService);
+		if (enabled) {
+			showOnStartup.setAttribute('checked', 'checked');
+		}
+
+		if (guidedTourEnabled) {
+			const adsHomepage = document.querySelector('.ads_homepage');
+			const guidedTourNotificationContainer = document.createElement('div');
+			const p = document.createElement('p');
+			const b = document.createElement('b');
+			const icon = document.createElement('div');
+			const containerLeft = document.createElement('div');
+			const containerRight = document.createElement('div');
+			const startTourBtn = document.createElement('a');
```

Comment:
We should really be trying to use the core UI components such as https://github.com/microsoft/azuredatastudio/blob/master/src/sql/base/browser/ui/button/button.ts#L14 instead of manually creating the elements

Reply:
Hey @Charles-Gagnon I believe I was told when I was working on the welcome page, that I wouldn't be able to use core components on the welcome page. The guided tour exists on the welcome page? Do you have any input? Thank you!

Reply:
Who told you that? There shouldn't be any reason you couldn't do that - the core components like the button I linked you just create the HTML themselves. It's basically doing what you're doing but : 

1. Handles a lot of the boilerplate stuff like creating all the items and making sure they're laid out correctly
2. Hooks up correctly to theming/events
3. Keeps code consolidated so we don't have a hundred different "button" implementations floating around everywhere

Reply:
I believe this has been fixed